docs/enterprise/index.md

```diff
@@ -104,20 +108,21 @@ Public and private Ethereum networks might need specific features required by ne
 ### Permissioning {#permissioning}
 
 - [PegaSys Permissioning Contracts](https://github.com/PegaSysEng/permissioning-smart-contracts)
-- [Quorum Permissioning solution](https://github.com/jpmorganchase/quorum/wiki/Security)
+- [Quorum Permissioning solution](http://docs.goquorum.com/en/latest/Permissioning/Overview/)
 
 ### Privacy {#privacy}
 
 - [Ernst & Young's ‘Nightfall'](https://github.com/EYBlockchain/nightfall)
   _More information [here](https://bravenewcoin.com/insights/ernst-and-young-rolls-out-'nightfall-to-enable-private-transactions-on)_
 - [PegaSys' Orion](https://docs.pantheon.pegasys.tech/en/stable/Concepts/Privacy/Privacy-Overview/)
   _More information [here](https://pegasys.tech/privacy-in-pantheon-how-it-works-and-why-your-enterprise-should-care/)_
-- [Quorum's Tessera](https://docs.goquorum.com/en/latest/Privacy/Tessera/Tessera/) _More information [here](https://github.com/jpmorganchase/tessera/wiki/How-Tessera-works)_
+- [Quorum's Tessera](https://docs.goquorum.com/en/latest/Privacy/Tessera/Tessera/) _More information [here](http://docs.goquorum.com/en/latest/Privacy/Lifecycle-of-a-private-transaction/)_
 
 ### Security {#security}
 
 - [Clef](https://github.com/ethereum/go-ethereum/tree/master/cmd/clef) _used to sign transactions and data and is meant as a replacement for geth’s account management_
 - [EthSigner](https://gitter.im/PegaSysEng/EthSigner) _A transaction signing application to be used with a web3 provider_
+- [Quorum Security Framework](http://docs.goquorum.com/en/latest/Security/Framework/Overview/) _Provides an in-depth account of Quorum and Decentralized Applications (dApps) security best practices_
```

Comment:
I'm a skeptical of adding this. There's not much content on it - primarily a list of "references" that links out to very broad resources (e.g. geth & quorum wikis, solidity docs, EEA technical docs). Where is the framework itself? Is that in the "Quorum Network" & "Decentralized App" sections? If so, could you please add a bit more of a summary & include links to those sections? It doesn't feel obvious where I'm supposed to navigate from this page in order to view the "Quorum Security Framework".

Reply:
many thanks for reviewing the changes and suggesting improvements. I have made the updates accordingly. thanks

Reply:
@drequinox thanks! But I don't seeing anything that's changed on http://docs.goquorum.com/en/latest/Security/Framework/Overview/ - am I missing something?

Reply:
many thanks, I understand the concern, I have removed the link to the security framework.